addons/mail/static/src/new/core/composer_model.js

```diff
@@ -55,6 +55,10 @@ export class Composer {
     update(data) {
         if ("textInputContent" in data) {
             this.textInputContent = data.textInputContent;
+            // place cursor at end of the content
```

Comment:
The comment seems a bit overkill, also this part of the code doesn't really place the cursor at the end as its just saving a state in model.

Also I don't really understand why this is in model generic update rather in the method actually triggering an edit.